custom-script/utils.py

```diff
@@ -38,15 +73,15 @@ def parse_subtree(node, brackets=True):
     return result
 
 
-def parse_one_tree(root, index):
+def parse_one_tree(root: Dict, index: int) -> str:
     return f"""
-function pt{index}({MODEL_INPUTS_STR}) {{ 
+function pt{index}({MODEL_INPUTS_STR}) {{
    return {parse_subtree(root, brackets=False)};
 }}
 """
 
 
-def parse_trees(trees):
+def parse_trees(trees: List) -> str:
     tree_functions = "\n".join([parse_one_tree(tree["tree_structure"], idx) for idx, tree in enumerate(trees)])
```

Comment:
does `join` work on generators? maybe you can get rid of brackers

Reply:
done